drivers/i2c/i2c_ite_enhance.c

```diff
@@ -481,15 +524,12 @@ static int i2c_transaction(const struct device *dev)
 	return 0;
 }
 
-static int i2c_enhance_transfer(const struct device *dev, struct i2c_msg *msgs,
-				uint8_t num_msgs, uint16_t addr)
+static int i2c_enhance_transfer(const struct device *dev, struct i2c_msg *msgs)
```

Comment:
Does this function also need to block standby? Or is CQ mode special in some way?

Reply:
No, PIO mode can still enter cpu idle. The EC processor cannot be in the cpu idle during the transactions with the CQ mode(DMA mode).